cpp/include/raft/cudart_utils.h

```diff
@@ -101,6 +105,15 @@ struct cuda_error : public raft::exception {
              __FILE__, __LINE__, cudaGetErrorString(status));             \
     }                                                                     \
   } while (0)
+#endif
+
+/**
+ * Alias to raft scope for now.
+ * TODO: Rename original implementations in 22.04 to fix
+ * https://github.com/rapidsai/raft/issues/128
+ */
+#define RAFT_CUDA_CHECK(call) CUDA_CHECK(call)
```

Comment:
I would have done this in reverse.  That is, I would have conditionally defined `CUDA_CHECK` as a call to `RAFT_CUDA_CHECK` which would make the next change smaller.  But this way is fine too.